app.py

```diff
@@ -99,6 +99,28 @@ def logout():
     flash("You are Logged out","danger")
     return redirect(url_for('login'))
 
+@app.route("/imageUpdate",methods=["POST"])
+def imageUpdate():
```

Comment:
this method is similar to def Upload() below, any reason you reproduced it again?

Also this method expects a template to be returned in the last line.

Reply:
actually, this method provides a route to files that can be uploaded from other nodes. i would be removing last line where is expects to return a template.